### PR TITLE
Add sku step and support standard sku in adv create

### PR DESCRIPTION
**Pull request by nturinski**

### CODE REVIEW

File: src/commands/createStaticWebApp/SkuListStep.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+export class SkuListStep extends AzureWizardPromptStep<IStaticWebAppWizardContext> {
+    public async prompt(wizardContext: IStaticWebAppWizardContext): Promise<void> {
+        const skus: IAzureQuickPickItem<WebSiteManagementModels.SkuDescription>[] = SkuListStep.getSkus().map(s => {
+            return {
+                label: nonNullProp(s, 'name'),
+                data: s
+            };
+        });
+
+        const placeHolder = localize('selectSku', 'Select a sku');
```
Comment (wwlorey) [inline src/commands/createStaticWebApp/SkuListStep.ts:21]: nit: 
```suggestion
        const placeHolder: string = localize('selectSku', 'Select a sku');
```

File: src/commands/createStaticWebApp/SkuListStep.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        wizardContext.sku = (await wizardContext.ui.showQuickPick(skus, { placeHolder, suppressPersistence: true })).data;
+    }
+
+    public shouldPrompt(wizardContext: IStaticWebAppWizardContext): boolean {
+        return !wizardContext.sku;
+    }
+
+    public static getSkus(): WebSiteManagementModels.SkuDescription[] {
+        // seems incomplete, but only name/tier are necessary for the SiteEnvelope to pick the right sku
```
Comment (wwlorey) [inline src/commands/createStaticWebApp/SkuListStep.ts:30]: Why do you say it seems incomplete?

**Review by wwlorey — APPROVED** [review]
I did some local testing too and it's looking good 👍

File: src/commands/createStaticWebApp/SkuListStep.ts (hunk 2 above)
Comment (nturinski) [reply src/commands/createStaticWebApp/SkuListStep.ts:30]: Ah, I just meant that the SkuDescription has a lot more properties.  And in azuretools/appservice, the SKUs that we hardcode have a lot more properties filled out.

**Pull request merged**